### PR TITLE
BDRE-231: adding color for process failed and running statuses

### PR DESCRIPTION
**Pull request by raiashu92**

### CODE REVIEW

**Comment by raiashu92** [comment]
@bdrekapil @Prem1993 check and merge.

File: md-dao/src/main/java/com/wipro/ats/bdre/md/dao/InstanceExecDAO.java [hunk 1]
```diff
@@ -178,17 +178,22 @@ public void delete(Long id) {
                 List<InstanceExec> instanceExecList=instanceExecListCriteria.list();
                 long sumTime=0;
                 long currentTime=0;
+                boolean processRunning = false;
+                int stateOfProcess = 0;
                 int total=instanceExecList.size();
                 for (InstanceExec instanceExec:instanceExecList)
                 {
                     if(instanceExec.getEndTs()!=null){
```
Comment (bdre-bda) [inline md-dao/src/main/java/com/wipro/ats/bdre/md/dao/InstanceExecDAO.java]: MINOR Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)

**Comment by bdre-bda** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by bdre-bda** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by raiashu92** [comment]
@Prem1993 @bdrekapil missing colors for Running and failed statuses have been added in the legend, check and merge

File: md-dao/src/main/java/com/wipro/ats/bdre/md/dao/InstanceExecDAO.java [hunk 2]
```diff
@@ -178,17 +178,24 @@ public void delete(Long id) {
...
                 long sumTime=0;
                 long currentTime=0;
+                boolean processRunning = false;
+                int stateOfProcess = 0;
+                int running = 0;
+                int failed = 0;
                 int total=instanceExecList.size();
                 for (InstanceExec instanceExec:instanceExecList)
                 {
                     if(instanceExec.getEndTs()!=null){
```
Comment (bdre-bda) [inline md-dao/src/main/java/com/wipro/ats/bdre/md/dao/InstanceExecDAO.java:188]: MINOR Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)

**Comment by bdre-bda** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by bdre-bda** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by raiashu92** [comment]
@bdrekapil pushed new changes, check and merge

**Comment by bdre-bda** [comment]
SonarQube analysis reported 1 issue:
- MINOR 1 minor

Watch the comments in this conversation to review them.

**Pull request merged**